src/sample/Controller.java

```diff
@@ -1,18 +1,50 @@
 package sample;
 
+import javafx.application.Platform;
 import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.chart.CategoryAxis;
+import javafx.scene.chart.LineChart;
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.XYChart;
 import javafx.scene.control.Button;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+import javafx.stage.Stage;
+import javafx.scene.paint.Color;
+
+import javax.imageio.*;
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.awt.image.BufferedImage;
+import java.io.*;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.ResourceBundle;
 
 public class Controller {
 
     //private declarations
     @FXML
-    private Button runBtn;
+    private Button speakBtn;
+    @FXML
+    private Button gatherBtn;
+    @FXML
+    private Label welcome;
     //end private declarations
 
     //public methods
-    public void onRunClicked(){
-        System.out.println("Hello.");
+    public void onGatherClicked(){
+        System.out.println("Gather mode activated");
     }
+    public void onSpeakPressed() {System.out.println("Speaking");}
```

Comment:
I suggest changing this method name to onSpeakClicked() to maintain naming convention.